inference/core/utils/image_utils.py

```diff
@@ -47,6 +47,15 @@ def load_image_rgb(value: Any, disable_preproc_auto_orient: bool = False) -> np.
     return np_image
 
 
+def load_image_bgr(value: Any, disable_preproc_auto_orient: bool = False) -> np.ndarray:
+    np_image, is_bgr = load_image(
+        value=value, disable_preproc_auto_orient=disable_preproc_auto_orient
+    )
+    if not is_bgr:
+        np_image = cv2.cvtColor(np_image, cv2.COLOR_BGR2RGB)
```

Comment:
what is this function supposed to do?

`cv2.COLOR_BGR2RGB` sounds like its converting it from BGR to RGB,
but the if sounds like this is only being done if the if image is BGR already?

Reply:
BGR2RGB just transposes the final channels (same as `np_image = np_image[:, :, ::-1]`)

so in the event that the returned image is not bgr, we transpose the final channels, making it bgr